torchsynth/module.py

```diff
@@ -568,14 +563,20 @@ class CrossfadeKnob(SynthModule):
     ]
 
 
-class NoteOnButton(SynthModule):
+class Keyboard(SynthModule):
```

Comment:
Call it MonophonicKeyboard?

Reply:
Close this issue https://github.com/turian/torchsynth/issues/117 when you merge

Reply:
Sounds good